src/spacewar/ui/icons.cljc

```diff
@@ -25,7 +25,7 @@
   (q/fill 0 0 0 150)
   (q/no-stroke)
   (q/ellipse-mode :center)
-  (q/arc 0 0 30 30 0 (age-angle age) :pie))
+  (q/arc 0 0 30 30 0 (age-angle age) #?(:clj :pie)))
```

Comment:
This last argument is not supported in the Quil ClojureScript API, so is conditionally excluded here.